openmc/mgxs/mgxs.py

```diff
@@ -1547,7 +1546,7 @@ def tallies(self):
         if self._tallies is None:
 
             # Create a list of scores for each Tally to be created
-            scores = ['flux', 'total', 'scatter-P1']
+            scores = ['flux', 'total', 'scatter-1']
```

Comment:
Hmmmm, thats possible.  Will this path also explode when more complex transport corrections are applied?  Is there something the Library can do for us instead of leaving it up to the Mgxs objects themselves?  That might not make much sense but its getting late for me.